crates/iceberg-ext/src/catalog/rest/table.rs

```diff
@@ -99,11 +105,104 @@ pub struct CommitTransactionRequest {
     pub table_changes: Vec<CommitTableRequest>,
 }
 
+#[must_use]
+pub fn create_etag(text: &str) -> String {
+    let hash = xxh3_64(text.as_bytes());
+    format!("\"{hash:x}\"")
+}
+
 #[cfg(feature = "axum")]
-impl_into_response!(LoadTableResult);
+impl IntoResponse for LoadTableResult {
+    fn into_response(self) -> axum::http::Response<axum::body::Body> {
+        // self.metadata_location
+        let metadata_location = self.metadata_location.clone().unwrap_or_default();
+        let etag = create_etag(&metadata_location);
+
+        let mut header = HeaderMap::new();
+        header.insert(header::ETAG, etag.parse().unwrap());
```

Comment:
We should handle the error gracefully and not panic. Lets use `tracing::error` to log the error and not return anything.